src/components/InfiniteHits.vue

```diff
@@ -85,13 +85,18 @@ export default {
         return items;
       },
     },
+    cache: {
+      type: Object,
+      default: undefined,
+    },
   },
   computed: {
     widgetParams() {
       return {
         showPrevious: this.showPrevious,
         escapeHTML: this.escapeHTML,
         transformItems: this.transformItems,
+        cache: this.cache,
```

Comment:
Do we want to test that it forwards the `cache` option to the connector? (I thought we did for all widgets.)

Reply:
I don't think we test every forward, since usually the test & implementation is written at the same time, so it will have the same mistake (if any) right?

Reply:
That's mainly to test future unintended changes and regressions. If we somehow edit/remove this line and that the `cache` feature doesn't work, no tests will tell us so.

(I'm pointing this out because I thought we did for all widgets — apparently we don't.)

Reply:
If so, what if we test if this component exposes a correct `widgetParams`? I think it's `createWidgetMixin`'s job to forward widget params to the connector and "is it forwarded correctly?" needs to be tested there.

If we want to keep the contract of what needs to be passed, then we could just test snapshots of the widgetParams.